src/qibocal/tree/task.py

```diff
@@ -21,16 +21,32 @@ class Output(ABC):
 class Task:
     operation: Operation
     parameters: Parameters
-    requirements: dict
+    _requirements: dict
+    _ready: bool
+
+    @property
+    def ready(self):
+        return self._ready
```

Comment:
Actually, this is fine to be a property, but it should be:
```suggestion
    @property
    def ready(self):
        return all(self.requirements.values())
```